zzupy/aio/app/eas/undergrad.py

```
@@ -0,0 +1,378 @@
+        Returns:
+            TeachingWeek: 教学周
+
+        Raises:
+            ParsingError: 如果响应解析失败
+            NetworkError: 如果网络请求失败
+            ValueError: 如果教学周序数不正确
+            OperationError: 如果服务器发生异常
+        """
+        logger.info("尝试获取第 {} 教学周...", week)
+        if week < 1:
+            raise ValueError("教学周序数不可小于 1")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)